src/wavesurfer.ts

```diff
@@ -120,6 +120,8 @@ export type WaveSurferEvents = {
   zoom: [minPxPerSec: number]
   /** Just before the waveform is destroyed so you can clean up your events */
   destroy: []
+  /** When the `container` is updated and wavesurfer remounts to it */
+  containerRemount: []
```

Comment:
A new event seems like an overkill for this. The setOptions operation is synchronous so the caller can just react to it immediately after the call.

Reply:
Removed the event.